ide/languages.hcl/manifest.mf

```diff
@@ -3,4 +3,5 @@ OpenIDE-Module: org.netbeans.modules.languages.hcl
 OpenIDE-Module-Layer: org/netbeans/modules/languages/hcl/layer.xml
 OpenIDE-Module-Localizing-Bundle: org/netbeans/modules/languages/hcl/Bundle.properties
 OpenIDE-Module-Specification-Version: 1.1
+OpenIDE-Module-Java-Dependencies: Java > 11
```

Comment:
Is this `>11` correct? Should it be `Java >= 11` instead?

Reply:
it looks weird, but is correct :)
https://github.com/search?q=repo%3Aapache%2Fnetbeans+OpenIDE-Module-Java-Dependencies%3A+Java+%3E+11&type=code

I believe there is no `>=`. `>` here basically means greater equals